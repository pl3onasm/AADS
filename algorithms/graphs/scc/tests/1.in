8

0 1
1 4
4 0
4 5
5 6
6 5
6 7
7 7
1 2
2 3
3 2
3 7
2 6
1 5


Source
======
CLRS‚Å¥ Figure 20.9
a = 0
b = 1
c = 2
d = 3
e = 4
f = 5
g = 6
h = 7
