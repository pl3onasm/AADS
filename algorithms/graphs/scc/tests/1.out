
--------------------
 Graph
 Directed
 Unweighted
 Vertices: 8
 Edges: 14
--------------------

   a[1]: b
   b[3]: f, c, e
   c[2]: g, d
   d[2]: h, c
   e[2]: f, a
   f[1]: g
   g[2]: h, f
   h[1]: h


Strongly connected components
-----------------------------
1: {h}
2: {g, f}
3: {c, d}
4: {b, a, e}